cmake/toolchains/aarch64-linux-gnu.cmake

```diff
@@ -0,0 +1,30 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
+
+set(CMAKE_C_COMPILER "aarch64-linux-gnu-gcc")
+set(CMAKE_CXX_COMPILER "aarch64-linux-gnu-g++")
+set(CMAKE_LINKER "aarch64-linux-gnu-ld")
+
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+
+set(CMAKE_C_FLAGS "-march=armv8-a")
+set(CMAKE_CXX_FLAGS "-march=armv8-a")
+
+# cache flags
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}" CACHE STRING "c flags")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}" CACHE STRING "c++ flags")
+
+set(CMAKE_INSTALL_PREFIX install)
+
+set(BUILD_PERF_TESTS OFF)
+set(BUILD_TESTS OFF)
+
+set(MMDEPLOY_BUILD_SDK ON)
+set(MMDEPLOY_BUILD_EXAMPLES ON)
+set(MMDEPLOY_BUILD_SDK_PYTHON_API OFF)
+set(MMDEPLOY_SHARED_LIBS OFF)
+set(MMDEPLOY_TARGET_DEVICES "cpu")
+set(MMDEPLOY_TARGET_BACKENDS "ncnn")
```

Comment:
As there are other .cmake files for backends like rknn, if the toolchain is only for ncnn, shall we specify the file name with ncnn?

Reply:
done.